fashion-messaging-app-ui/src/components/Search/Search.jsx

```diff
@@ -1,18 +1,32 @@
+
 import React, { useState } from "react";
 import "./Search.css";
 import { FaSearch} from "react-icons/fa";
 
-export default function Search() {
+export default function Search({ onSearch }) {
+  const [searchQuery, setSearchQuery] = useState("");
+
+  const handleSearchInput = event => {
+    setSearchQuery(event.target.value);
+  };
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    onSearch(searchQuery);
+  };
+
   return (
     <div className="searchBtn">
-      <form>
-        <input
-          className="searchInput"
-          type="text"
-          placeholder="Search for outfits!"   
-        />
-        <button type="submit" className="submitBtn"><FaSearch/></button>
-      </form>
+       <form onSubmit={handleSubmit}>
+      <input
+       className="searchInput"
+        type="text"
+        value={searchQuery}
+        onChange={handleSearchInput}
+        placeholder="Search your outfits!"
+      />
+      <button type="submit"className="submitBtn"><FaSearch/></button>
```

Comment:
nit: could you add spaces between "submit" and className?

Reply:
> nit: could you add spaces between "submit" and className?

Okay!